src/main/java/eu/europa/ted/eforms/sdk/component/SdkComponentDescriptor.java

```diff
@@ -50,7 +50,7 @@ public T createInstance(Object... initArgs) throws InstantiationException {
           .collect(Collectors.toList())
           .toArray(new Class[0]);
 
-      logger.trace("Creating an instance of [{}] using constructor with parameter types: {}",
+      logger.debug("Creating an instance of [{}] using constructor with parameter types: {}",
```

Comment:
Having an entry at debug level for each instanciation seems like a lot.
I think this should stay at the "trace" level.

Reply:
I hesitated for this one but OK